### PR TITLE
[WIP] Add `--aev` flag on `aea config` command

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR enables the usage of `--aev` flag on `aea config` command to support 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/test_tools/test_cases.py [hunk 1]
```diff
@@ -146,6 +146,7 @@ def set_config(
             str(value),
             "--type",
             type_,
+            "--aev",
```
Comment (angrybayblade) [inline aea/test_tools/test_cases.py:149]: @8ball030 We use `--aev` by default on the base test class

**Review by 8ball030 — APPROVED** [review]
Great, this should mean that deployable can be removed alot lot easier

**Review by Karrenbelt — APPROVED** [review]
LGTM

File: aea/test_tools/test_cases.py (hunk 1 above)
Comment (8ball030) [reply aea/test_tools/test_cases.py:149]: Cool, I think its important we dont set this by default. otherwise that will lead to a bad time.

Comment (angrybayblade) [reply aea/test_tools/test_cases.py:149]: I'll make this as a argument

File: setup.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def get_all_extras() -> Dict:
 
 if os.name == "nt" or os.getenv("WIN_BUILD_WHEEL", None) == "1":
     if sys.version_info[1] > 9 and sys.version_info[0] == 3:
-        base_deps.append("pywin32==305")
+        base_deps.append("pywin32==304")
```
Comment (angrybayblade) [inline setup.py:69]: `pywin32==305` does not have a release: https://pypi.org/project/pywin32/

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**